services/agents/registry.go

```diff
@@ -458,30 +458,32 @@ func (r *Registry) CheckConnectionToService(ctx context.Context, service *models
 	switch service.ServiceType {
 	case models.MySQLServiceType:
 		request = &agentpb.CheckConnectionRequest{
-			Type: inventorypb.ServiceType_MYSQL_SERVICE,
-			Dsn:  agent.DSN(service, 2*time.Second, ""),
+			Type:    inventorypb.ServiceType_MYSQL_SERVICE,
+			Dsn:     agent.DSN(service, 2*time.Second, ""),
+			Timeout: ptypes.DurationProto(3 * time.Second),
```

Comment:
maybe we should extract this timeout and timeout above to consts, what do you think?

Reply:
I don't think they will be constants for long, to be honest. For example, it is likely we will use higher values for remote services.